### PR TITLE
Add default value for Wall.heat_spec

### PR DESCRIPTION
**Pull request by angranl-flex**
Set the default heat_spec of Wall to heatFlux=0 now

### CODE REVIEW

File: flow360/component/simulation/models/surface_models.py [hunk 1]
```diff
@@ -165,7 +165,7 @@ class Wall(BoundaryBase):
         None, description="Prescribe a tangential velocity on the wall."
     )
     heat_spec: Optional[Union[HeatFlux, Temperature]] = pd.Field(
-        None,
+        HeatFlux("0"),
```
Comment (benflexcompute) [inline flow360/component/simulation/models/surface_models.py:168]: Can we not use expression here? Why do we use expression when we are trying to set a constant value?

Comment (angranl-flex) [reply flow360/component/simulation/models/surface_models.py]: Sure, use a constant value now.

File: flow360/component/simulation/models/surface_models.py [hunk 2]
```diff
@@ -169,8 +170,9 @@ class Wall(BoundaryBase):
     velocity: Optional[VelocityVectorType] = pd.Field(
         None, description="Prescribe a tangential velocity on the wall."
     )
+    # pylint: disable=no-member
     heat_spec: Optional[Union[HeatFlux, Temperature]] = pd.Field(
```
Comment (benflexcompute) [inline flow360/component/simulation/models/surface_models.py:174]: Remove Optional

**Review by benflexcompute — COMMENTED** [review]
Approved but let's wait for front end to make change and then merge.

**Review by benflexcompute — APPROVED** [review]

**Pull request merged**